cli/commands.go

```diff
@@ -238,6 +239,8 @@ func runDaemon(d *app.MenderDaemon) error {
 			} else if s == syscall.SIGUSR2 {
 				log.Debug("SIGUSR2 signal received.")
 				d.ForceToState <- app.States.InventoryUpdate
+			} else if s == syscall.SIGTERM {
+				d.StopDaemon()
```

Comment:
Have you tested this? It doesn't look like this breaks out of any sleep, it just sets a flag.

Reply:
It breaks out of sleep on the following line, no?

Reply:
Oh, lol! You're right. The codecov box hid it from me!